README.md

```diff
@@ -15,6 +15,26 @@ like the ones who try to install `snap` packages.
 
 Here you can find the (installers)[https://github.com/actions/virtual-environments/tree/main/images/linux/scripts/installer] here.
 
+## Configurations
+
+Github Actions Runner can be configured dynamically using env vars with the following prefix `ACTIONS_RUNNER_INPUT_`.
```

Comment:
Oh really? I wasn't aware of that. Do you have a reference to some docs?

Reply:
Yes they seems to be undocumented but digging inside the code I found them, you can find a couple of links on the issue.
I guess I’ll open a PR to the runner project to  document this behavior.